config/initializers/action_cable_acts_as_tenant.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
```

Comment:
Can we add a reference to this? `https://github.com/ErwinM/acts_as_tenant/pull/280/files`